topo/node/cptx/cptx.go

```diff
@@ -96,6 +96,8 @@ func (n *Node) SpawnCLIConn(ns string) error {
 		scraplibase.WithAuthBypass(true),
 		// disable transport timeout
 		scraplibase.WithTimeoutTransport(0),
+		// for committing a very large config
+		scraplibase.WithTimeoutOps(300 * time.Second),
```

Comment:
Style nit:

I think it would be better to define a var block at the top of the file with the different timeout durations used in this file.

i.e.

var (
    waitForCLITimeout = 500 * time.Second
    scrapliDriverTimeout = 300 * time.Second
)

and then referenced as needed. IMO it would add clarity

Reply:
Agree (y). I have taken care of it in the next commit.